build.gradle.kts

```diff
@@ -24,13 +24,20 @@ dependencies {
     testImplementation("net.jodah:concurrentunit:0.4.6")
 }
 
-val gitVersion: groovy.lang.Closure<Any> by extra
-
-group = "com.staffbase.quartz-redis-jobstore"
-version = gitVersion()
+// these values don't matter, they're changed by jitpack when building
+group = "com.github.Staffbase"
+version = "1.0.0-SNAPSHOT"
 
 java.sourceCompatibility = JavaVersion.VERSION_1_8
 
 tasks.withType<JavaCompile> {
     options.encoding = "UTF-8"
 }
+
+publishing {
```

Comment:
we need this for jitpack to understand what we want to do

We don't currently package the sources, I didn't want to set that up too right now